tests/CMakeLists.txt

```diff
@@ -1,9 +1,14 @@
-cmake_minimum_required (VERSION 3.0)
+cmake_minimum_required (VERSION 2.8)
 
 project (lwm2munittests)
 
 include(${CMAKE_CURRENT_LIST_DIR}/../core/wakaama.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/../examples/shared/shared.cmake)
+
+if (CMAKE_COMPILER_IS_GNUCC)
+    set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} -std=gnu99)
```

Comment:
With CMake 2.8.12.2 and GCC 4.9.3, I get compiler errors without this:
/mnt/repos/wakaama-ri/tests/convert_numbers_test.c: In function ‘test_lwm2m_PlainTextToInt64’:
/mnt/repos/wakaama-ri/tests/convert_numbers_test.c:40:5: error: ‘for’ loop initial declarations are only allowed in C99 or C11 mode
     for (int i = 0 ; tests[i] != NULL ; i++)

From: David Navarro [mailto:notifications@github.com]
Sent: Wednesday, June 15, 2016 2:36 AM
To: eclipse/wakaama <wakaama@noreply.github.com>
Cc: Bert Kleewein <bertk@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [eclipse/wakaama] COAP over TCP based on draft-ietf-core-coap-tcp-tls-01 (#174)


In tests/CMakeLists.txt<https://github.com/eclipse/wakaama/pull/174#discussion_r67129868>:

>

>  project (lwm2munittests)

>

>  include(${CMAKE_CURRENT_LIST_DIR}/../core/wakaama.cmake)

>  include(${CMAKE_CURRENT_LIST_DIR}/../examples/shared/shared.cmake)

> +

> +if (CMAKE_COMPILER_IS_GNUCC)

> +    set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} -std=gnu99)

Why do you need this ?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/eclipse/wakaama/pull/174/files/fa9abf1226e7d4d61dbc38bada0b87cf300d8f87#r67129868>, or mute the thread<https://github.com/notifications/unsubscribe/AHqCQiNqOpqo6_f2GtvILhjQ7n2CYnMOks5qL8eQgaJpZM4I1z-O>.